nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -77,22 +51,43 @@ class FastBiasCorrection(Algorithm):
         the sub-graph and further quantization output calculation;
         - in the end we corrects the original bias by the difference (shift)
         between floating-point and quantized outputs.
-
-    :param number_samples: The number of the samples for the statistics collection.
-    :param threshold: The magnitude threshold that regulates the application of the shift.
-    :param nncf_graph: NNCFGraph class for the algorithm.
     """
 
-    def __init__(self, parameters: FastBiasCorrectionParameters) -> None:
+    def __init__(self, 
+                 subset_size: int = 100,
+                 threshold: float = FAST_BIAS_CORRECTION_THRESHOLD,
+                 apply_for_all_nodes: bool = False,
+                 inplace_statistics: bool = True,
+                 backend_params: Optional[Dict[str, Any]] = None):
         """
-        :param parameters: The instance of the FastBiasCorrectionParameters.
+        :param subset_size: Size of a subset for the statistics collection,
+            defaults to 100.
+        :param threshold: The magnitude threshold that regulates the application of the
+            shift. Magnitude calculates as the maximum of the absolute ratio of the
+            shift to the original bias value. If the calculated value is less than the
+            threshold, the shift will apply to the bias, defaults to 2.
+        :param apply_for_all_nodes: If True, then the bias correction be applied to all
+            quantized nodes, if the node has no bias then a bias node will be inserted,
+            and if False, then the bias correction will only be applied to quantized
+            nodes that have a bias.
+        :param inplace_statistics: Defines wheather to calculate quantizers statistics
+            by backend graph operations or by default Python implementation, defaults
+            to True.
+        :param backend_params: Backend specific parameters.
         """
         super().__init__()
-        self.number_samples = parameters.number_samples
-        self.threshold = parameters.threshold
-        self.inplace_statistics = parameters.inplace_statistics
+        self.subset_size = subset_size
+        self.threshold = threshold
+        self.apply_for_all_nodes = apply_for_all_nodes
+        self.inplace_statistics = inplace_statistics
+        self.backend_params = backend_params
         self.nncf_graph = None
         self._backend_entity = None
+        
+        if self.apply_for_all_nodes:
+            raise RuntimeError(
+                'FastBiasCorrection algorithm from the OpenVINO backend does not '
+                'support apply_for_all_nodes=True yet')
```

Comment:
Is this backend-specific runtime error in common code put in right position?

Reply:
This is not backend-specific runtime error. Fixed message.